index.html

```diff
@@ -257,12 +276,21 @@
         <a title="user agent">User agents</a> MUST NOT provide Push API access to <a title=
         "webapp">webapps</a> without the <a>express permission</a> of the user. <a title=
         "user agent">User agents</a> MUST acquire consent for permission through a user interface
-        for each call to the <code>subscribe()</code> method, unless a prearranged trust
-        relationship applies.
+        for each call to the <code>subscribe()</code> method, unless a previous permission grant
+        has been persisted, or a prearranged trust relationship applies. Permissions that are
```

Comment:
A prearranged trust relationship can also be part of an EULA or something like that. The use of persisted permission grants is by far the most common thing for web APIs, so it makes sense to call it out, for clarity. Finally, whether a persisted grant is "prearranged" seems open to interpretation. The new phrasing seems clearer to me.